README.rst

```diff
@@ -119,12 +280,210 @@ Senza Definition
+parameters-section-structure.html>`_ such as ``AllowedPattern``, ``AllowedValues``,
+``MinLength``, ``MaxLength``, etc. Senza itself will not enforce these,
+but CloudFormation will evaluate the generated template and raise an exception
+if any of the properties are not met. For example:
+
+.. code-block:: bash
+
+    $ senza create example.yaml 3 example latest mint-bucket "Way too long greeting"
+    Generating Cloud Formation template.. OK
+    Creating Cloud Formation stack hello-world-3.. EXCEPTION OCCURRED: An error occurred (ValidationError) when calling the CreateStack operation: Parameter 'GreetingText' must contain at most 15 characters
+    Traceback (most recent call last):
+    [...]
+
+Using the ``Default`` attribute, you can give any parameter a default value.
+If a parameter was not specified on the command line (either as positional or
+named), the default value is used. We suggest putting all default-value
+parameters at the bottom of your parameter definition list. Otherwise, there will be no way to map them to
+proper positions, and you'll have to specify all the following
+parameters using a ``name=value``.
+
+There is an option to pass parameters from a file (the file needs to be formatted in .yaml):
+
+.. code-block:: bash
+
+    $ senza create --parameter-file parameters.yaml example.yaml 3 1.0-SNAPSHOT
+
+An example of a parameter file:
+
+.. code-block:: yaml
+
+   ApplicationId: example-app-id
+   MintBucket: your-mint-bucket
+
+You can also combine parameter files and parameters from the command line, but you can't name the same parameter twice. Also, the parameter can't exist both in a file and on the command line:
+
+.. code-block:: bash
+
+    $ senza create --parameter-file parameters.yaml example.yaml 3 1.0-SNAPSHOT Param=Example1
+
+AccountInfo
+================
+
+Senza templates offer the following properties:
+
+``{{AccountInfo.Region}}``: the AWS region where the stack is created. Ex: 'eu-central-1'. In many parts of a template, you can also use `{"Ref" : "AWS::Region"}`.
+
+``{{AccountInfo.AccountAlias}}``: the alias name of the AWS account. Ex: 'super-team1-account'.
+
+``{{AccountInfo.AccountID}}``: the AWS account id. Ex: '353272323354'.
+
+``{{AccountInfo.TeamID}}``: the team ID. Ex: 'super-team1'.
+
+``{{AccountInfo.Domain}}``: the AWS account domain. Ex: 'super-team1.net'.
+
+Mappings
+================
+
+Senza mappings are essentially key-value pairs, and behave just like `CloudFormation mappings <http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/mappings-section-structure.html>`_. Use mappings for ``Images``, ``ServerSubnets`` or ``LoadBalancerSubnets``. 
+
+An example:
+
+.. code-block:: yaml
+
+   Mappings:
+      Images:
+         eu-west-1:
+            MyImage: "ami-123123"
+   # (..)
+   Image: MyImage
+
+Senza Components
+================
+
+Configure all your Senza components in a list below the top-level "SenzaComponents" key. The structure is as follows:
```

Comment:
If you replace "SenzaComponents" with \`SenzaComponents\` (here and in the other places it is referenced) it will look nicer (`SenzaComponents`)